packages/integrations/vercel/README.md

```diff
@@ -401,6 +401,14 @@ When you opt in to this feature, there are few constraints to note:
 - Only `request` and `context` may be used to produce an `Astro.locals` object. Operations like redirects, etc. should be delegated to Astro middleware.
 - `Astro.locals` **must be serializable**. Failing to do so will result in a **runtime error**. This means that you **cannot** store complex types like `Map`, `function`, `Set`, etc.
 
+### Node.js Version Support
+
+The `@astrojs/vercel` adapter supports specific Node.js versions for deploying your Astro project on Vercel. To view the supported Node.js versions, click on the settings tab for a project and scroll down to "Node.js Version" section. The supported versions are categorized as 'current', 'beta', and 'deprecated'. Each category represents the support status of that Node.js version:
+
+- **Current**: Fully supported and recommended for use.
+- **Beta**: Newer versions that are in beta. These are available for testing and early adoption.
+- **Deprecated**: Versions that are no longer recommended and scheduled for removal on a specific date.
```

Comment:
```suggestion
### Node.js Version Support

The `@astrojs/vercel` adapter supports specific Node.js versions for deploying your Astro project on Vercel. To view the supported Node.js versions on Vercel, click on the settings tab for a project and scroll down to "Node.js Version" section.

Check out the [Vercel documentation](https://vercel.com/docs/functions/serverless-functions/runtimes/node-js#default-and-available-versions) to learn more.
```
I think it makes more sense because this way, we do not need to keep this doc in sync with Vercel.